.ci/publish.jenkins

```diff
@@ -0,0 +1,150 @@
+#!groovy​
+
+// TODO: Move to a file and avoid modifying CI script
```

Comment:
We can also generate it dynamically by having a look at the `release/xxx` branches.